pkg/agent/openflow/pipeline.go

```diff
@@ -146,37 +147,40 @@ func (c *client) defaultFlows() (flows []binding.Flow) {
 		default:
 			flowBuilder = flowBuilder.Action().Drop()
 		}
-		flows = append(flows, flowBuilder.Done())
+		flows = append(flows, flowBuilder.Cookie(c.cookieAllocator.Request(cookie.Default).Raw()).Done())
```

Comment:
What about move the cookieID settings to flowBuilder constructor if every flow needs this property?

Reply:
IMO, explicitly setting cookie looks better. If you have strong opinion on it, I'm ok to change it also.

Reply:
The current code looks good to me